eng/config/PublishData.json

```diff
@@ -209,7 +209,7 @@
       "nugetKind": [ "Shipping", "NonShipping" ],
       "version": "3.9.*",
       "packageFeeds": "default",
-      "channels": [ "dev16.9p2" ],
+      "channels": [ "dev16.9p3" ],
```

Comment:
should master-vs-deps also switch to p4?

Reply:
Of course, how could anyone forget to do that
<details>
  <summary>‎</summary>
oops
</details>